app/services/repository_form.rb

```diff
@@ -3,9 +3,9 @@
 class RepositoryForm
   include ShallowAttributes
 
-  attribute :repo_path, String
+  attribute :path, String
```

Comment:
I'd left `repo_path` instead of `path`

Reply:
Repository model has "path" field instead of "repo_path" so it's consistency between the model and form